test.scss

```diff
@@ -0,0 +1,4 @@
+header{
```

Comment:
You can also remove this. We need to get your CSS cleaned up before we start to work in Sass